### DISCUSSION TITLE
How to post to Symbiota Support Discussions

### DISCUSSION THREAD

**Discussion by themerekat** [Symbiota/community-board#22] [Announcements]
We're here to help with your Symbiota-related needs. Please start or contribute to a discussion in one of our categories:

- **Problems/bugs** - bring up a problem you are having with your portal instance. Please mention the portal you are using when you start your discussion
- **Wish List** - contribute ideas for tools and developments to the Wish List section. If these suggestions are upvoted, certain tools may be converted into issues for further development.
- **Q&A** - ask for help with Symbiota, your collection, or digitization
- **Show & Tell** - describe tips, tricks, or developments from your Symbiota portal
- **Funding** - discuss ways of getting your favorite tools funded for development (there's only so much time to go around)

**Comment by arbolitoloco** [comment]
Also:
### Make sure you upvote the enhancements in the [Wish List](https://github.com/BioKIC/symbiota-docs/discussions/categories/wish-list) so that we can see what features are the most desired by the community:

![symbiota-docs-issue-upvote-demo](https://user-images.githubusercontent.com/17555247/140991007-d8cba365-b719-497f-86b6-6cc6fab88f83.gif)

**Comment by ljwalker** [comment]
Another tutorial for using Discussions can be found on our website: https://symbiota.org/symbiota-discussions/

**Comment by grrink** [comment]
Does anyone know how I can make a new password, when I don't have my old one?  The only way I can see to do it, is by using the old password, which, if you've lost your password, is not much help.

**Reply by themerekat** [reply]
@grrink just sent you a response to your email. Let me know if you don't receive it!

**Comment by tcbarry** [comment]
CCH2, problems/bugs: When I search a record logged in as an administrator from the collection management section the image appears at the usual spot (pre beta-testing), but the data appears lower on the screen so the data and image are no longer side by side. This requires extra scrolling to see the data. I already refreshed my cache yesterday. Browsers: Chrome & Firefox. Example: catalog number  DAV352666
Thank you, Teri Barry DAV

**Reply by themerekat** [reply]
Thanks @tcbarry ! This has been reported and logged here: https://github.com/BioKIC/Symbiota/issues/1321
And we're currently investigating a fix.

**Comment by tcbarry** [comment]
CCH2 beta test, problems/bugs:  When I search a record logged in as an administrator from the collection management section and no records match the query there is now no way to get back to the search or move back through the path. I have to open another tab and am still logged on.

**Reply by themerekat** [reply]
@tcbarry , try clicking the Open Record Search Form button!

**Reply by themerekat** [reply]
![image](https://github.com/BioKIC/symbiota-docs/assets/16688784/023667fe-d212-4d6a-abba-7ece3c9f76f9)

**Comment by tcbarry** [comment]
This is all I see. Nothing is clickable….

***@***.***

**Reply by themerekat** [reply]
@tcbarry , can you email help@symbiota.org? The email to GitHub didn't bring the image with it.

**Comment by fbungartz** [comment]
Programming priorities should not be determined by how popular a feature request is, but by its impact, i.e., of how it improves functionality like more intuitive data entry and less errors.

A good example is the determination tab. Few users are even aware of this functionality and it is common that users overwrite existing determinations on the occurrence tab. A feature to automatically adf a new determination record when overwriting existing IDs would much improve functionality and data safety, but likely not receive much support since many users might not even be aware of the concept of keeping a determination history.
